torch/_export/passes/replace_view_ops_with_view_copy_ops_pass.py

```diff
@@ -0,0 +1,59 @@
+from typing import Dict, List, Set, Type
+
+import torch
+from torch._ops import OpOverload, OpOverloadPacket
+from torch._export.pass_base import ExportPassBase
+
+
+__all__ = ["ReplaceViewOpsWithViewCopyOpsPass"]
+
+
+_NON_FUNCTIONAL_OPS_TO_FUNCTIONAL_OPS: Dict[OpOverload, OpOverload] = {
+    torch.ops.aten._unsafe_view.default: torch.ops.aten.view_copy.default,
+}
+
+# TODO (tmanlaibaatar) remove this after https://github.com/pytorch/pytorch/pull/100749
+_BLACK_LISTED_OPS: Set[OpOverloadPacket] = {
+    torch.ops.aten.sym_size,
+    torch.ops.aten.sym_stride,
+    torch.ops.aten.sym_numel,
+}
+
+def is_view_op(schema: torch._C.FunctionSchema) -> bool:
+    if len(schema.arguments) == 0:
+        return False
+    return schema.arguments[0].alias_info is not None
```

Comment:
This looks a little suspicious to me.. Are there other operators that may carry this alias_info? e.g. view, slice, expand..

Reply:
Yeah this is little sketchy. But @bdhirsh  mentioned it should be enough for our case :).

Reply:
And operators like slice should also be replaced with slice_copy.

Reply:
So this pass will only replace view with view_copy or can it also replace other operators with their copy variants?

Reply:
It will replace any op that aliases (essentially any view op) with copy variant.

Reply:
Ok, maybe we should change to something like replace_aliasing_ops_with_copy_variants_pass.py? because it's easy to be confused with the view op with the more general view ops such as expand, slice, transpose. Can we also add a comment to explain why checking "arguments[0].alias_info is not None" is sufficient in most cases.

Reply:
oh, this will probably also cause you to return true for inplace operators (which have mutable alias annotations). I think you want:
```
info = schema.arguments[0].alias_info
return info is not None and not info.is_write
```